.github/actions/install_syft_grype/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Install Syft & Grype
+description: |
+  Installs Syft & Grype.
+inputs:
+  versionSyft:
+    description: "Version of the operator-sdk to install"
+    default: "0.66.2"
+    required: false
+  versionGrype:
+    description: "Version of Grype to install"
+    default: "0.55.0"
+    required: false
+  os:
+    description: "Download build of Syft & Grype for operating system [linux, darwin]"
+    default: "linux"
+    required: false
+  arch:
+    description: "Download build of Syft & Grype for architecture [amd64, arm64]"
+    default: "amd64"
+    required: false
```

Comment:
Doesn't work for Darwin (RUNNER_OS returns `macOS`).

Reply:
Needs a translation layer (x64 -> amd64, macOS -> Darwin) but I'll add this quickly. Not really needed since Linux callers only use it currently (and there's not ARM runners), but uh, let's go for it I guess.

Reply:
Alright, new version should support Linux/macOS and amd64/arm64.